packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -255,6 +233,9 @@ const LayoutHeader = forwardRef((props, ref) => {
     siteData.pathPrefix
   );
 
+  // The siteContext is also being used to display the site context in algolias search results.
+  const siteContext = `${siteContextTitle} > ${props.siteTitle}`;
```

Comment:
I'm not sure if we can reproduce the exact style of the text inside of the top bar as it was before. For example the bold site title. Suggestions would be welcome :)

Reply:
Maybe a possible solution would be to leave untouched the current UI implementation and create a hidden div, with the  `site-context` id,  containing just the text you need and then select it using the algolia crawler selector

Reply:
We spoke about the implementation approach in the grooming and other locations,  I recommended a custom HTML head meta attribute using a name that is not in common use yet ( https://wiki.whatwg.org/wiki/MetaExtensions ), e.g. "commercetools:title-for-onsite-search" or whatever you think makes sense.